GooglePlayPlugins/com.google.play.billing/Samples/TrivialKart/README.md

```diff
@@ -0,0 +1,111 @@
+3. Following the [instructions](https://docs.unity3d.com/Manual/UnityIAPSettingUp.html) to Enable IAP service. 
+You may need to manually import the package at Assets/Plugins/UnityPurchasing/UDP.unitypackage and Assets/Plugins/UnityPurchasing/UnityIAP.unitypackage
+(The auto import doesn't work when there are errors in the scripts).
+4. Set up the Google Play Game Package Registry for Unity with [this method](https://developer.android.com/games/develop/build-in-unity#option_2_manually_edit_manifestjson).
+    Then install Google Play Billing Library at **Window > Package Manager**. Follow [this instruction](https://developer.android.com/google/play/billing/unity#plugin-build-settings) to configure build settings.
+    (note: you may need to restart Unity to make Google appear at the menu bar).
+
+   GO TO GOOGLE PLAY DEVELOPER CONSOLE
+5. Create an application on the Developer Console, available at https://play.google.com/apps/publish/.
+6. Copy the application's public key (a base-64 string). You can find this in the "Services & APIs" section under "Licensing & In-App Billing".
+
+    GO BACK TO UNITY EDITOR
+7. Added Google Public Key to Obfuscated encryption keys. Check https://docs.unity3d.com/Manual/UnityIAPValidatingReceipts.html for instructions
+(note: If you can't see Unity IAP under Window in menu bar, you may need to reinstall UDP at **Assets/UDPInstaller/Editor/UDP.unitypackage**).
+8. Sign the App by creating a new key store at **File > Build Settings > Player Settings > Publishing Settings**. You can follow [this instruction](https://answers.unity.com/questions/326812/signing-android-application.html)
+9. Build the game.
+
+    GO BACK TO GOOGLE PLAY DEVELOPER CONSOLE
+10. Upload your APK to Googole Play for Beta Testing. 
+
+11. Under Monetize -> Products, create In-app products (note: if you are using Classic Play Console, create MANAGED in-app items under In-app Products) with these IDs and prices:
+     
+      | Product ID   |  Price|
+      | :---:        | :---: |
+      | car_jeep     | $2.99 |
+      | car_kart     | $4.99 |
+      | five_coins   | $0.99 |
+      | ten_coins    | $1.99 |
+      |twenty_coints | $2.49 |
+      |fifty_coins   | $4.99 |
+
+     Fill the other fields. Set them to "Active".
+
+12. Create subscriptions with these IDs and prices: 
+
+     | Product ID   |  Price|
+     | :---:        | :---: |
+     | silver_subscription   | $1.99 |
+     | golden_subscription    | $4.99 |
+
+    Fill the other fields. Set them to "Active".
+
+13. Publish your APK to the Alpha channel. Wait 2-3 hours for Google Play to process the APK. If you don't wait for Google Play to process the APK, you might see errors where Google Play says that "this version of the application is not enabled for in-app billing" or something similar.
+
+14. Added Tester and License testing to your game on Play Console.
```

Comment:
nit: add a link to licensed tester since it's a play billing concept?

Reply:
This is an optional step that will make testing easier. It's still testable without doing this.